templates/zerver/app/left_sidebar.html

```diff
@@ -5,7 +5,7 @@
             <li class="top_left_all_messages top_left_row" title="{{ _('All messages') }} (a)">
                 <a href="#all_messages" class="home-link">
                     <span class="filter-icon">
-                        <i class="fa fa-home" aria-hidden="true"></i>
+                        <svg class="globe-web-icon" viewBox="0 0 495.9 496"><path d="M336.5,160C322,70.7,287.8,8,248,8s-74,62.7-88.5,152ZM152,256a626.79,626.79,0,0,0,3.3,64H340.6a626.79,626.79,0,0,0,3.3-64,626.79,626.79,0,0,0-3.3-64H155.3A626.79,626.79,0,0,0,152,256Zm324.7-96a248.52,248.52,0,0,0-158-141.6c24.4,33.8,41.2,84.7,50,141.6ZM177.2,18.4A248.36,248.36,0,0,0,19.3,160h108c8.7-56.9,25.5-107.8,49.9-141.6ZM487.4,192H372.7c2.1,21,3.3,42.5,3.3,64s-1.2,43-3.3,64H487.3a245.86,245.86,0,0,0,8.6-64,250.16,250.16,0,0,0-8.5-64ZM120,256c0-21.5,1.2-43,3.3-64H8.6a242.44,242.44,0,0,0,0,128H123.2C121.2,299,120,277.5,120,256Zm39.5,96C174,441.3,208.2,504,248,504s74-62.7,88.5-152ZM318.8,493.6A248.76,248.76,0,0,0,476.8,352h-108C360,408.9,343.2,459.8,318.8,493.6ZM19.3,352a248.52,248.52,0,0,0,158,141.6c-24.4-33.8-41.2-84.7-50-141.6Z" transform="translate(0 -8)"/></svg>
```

Comment:
![image](https://user-images.githubusercontent.com/25124304/95212704-9d339d00-080b-11eb-9066-1732afe22391.png)
<img width="790" alt="Screenshot 2020-10-06 at 7 32 26 PM" src="https://user-images.githubusercontent.com/25124304/95212737-a3297e00-080b-11eb-95bf-bf6bb9c8c7b4.png">

Reply:
Looks nice visually; do we need to also change something in the "Search" UI?